RedditSharp/Things/LiveUpdateEvent.cs

```diff
@@ -545,7 +539,7 @@ private void CommonInit(Reddit reddit, JToken json, IWebAgent webAgent)
             WebAgent = webAgent;
         }
 
-        private void SimpleAction(string url)
+        private new void SimpleAction(string url)
```

Comment:
https://github.com/CrustyJew/RedditSharp/blob/09168ddc12e7b1bbf5f76c956ca9034435ffb542/RedditSharp/Things/Thing.cs#L233

It's a protected member not private. Changing it to `protected override` should work just fine.